packages/request-manager/src/torIdentities.ts

```diff
@@ -31,6 +31,7 @@ export class TorIdentities {
                 port: this.torController.options.port,
                 userId: user,
                 password: password || user,
+                timeout,
```

Comment:
If I understand well the reason of introducing that `timeout` here is to have same in `cross-fetch` as in `SocksProxyAgent` in order to avoid inconsistencies in timeout in the same request.
I have being logging `timeout` here and I have only seen `undefined`, is it somewhere explicitly assigned or are you planning to?